### PR TITLE
Plasmamen ERT fix and updated sprites for CE/CMO Enviroclothing.

### PR DESCRIPTION
**Pull request by GuillaumePrata**
## About The Pull Request
Some long time ago I made a PR to add support for Plasmaman ERT so they don't spawn in human clothing and die.
Then I fucked up on my github stuff somewhere and that file got lost to the void while all others got in...
This fixes that, also adds some improved sprites and new outfits for the ERP team.

## Why It's Good For The Game
Not burning alive after getting your sweet Intern role is quite good!
Also better sprites following feedback.

## Changelog
:cl: Guillaume Prata and M. Joyce
fix: Fixes Plasmamen ERT not spawning properly
imageadd: Updated sprites for the CE and CMO Envirosuits and some small fixes on others.
/:cl:

### CODE REVIEW

**Review by Twaticus — APPROVED** [review]
since when did we even have these

**Comment by GuillaumePrata** [comment]
> since when did we even have these

August, before that Command Plasmaman would spawn with an Assistant Envirosuit if the Human only config was disabled.
And well... ERT Plasmaman would spawn with human gear and die...

File: code/modules/clothing/outfits/plasmaman.dm [hunk 1]
```diff
@@ -235,3 +235,35 @@
...
+/datum/outfit/plasmaman/party_janitor
+	name = "ERP Cleaning Plasmaman"
+
+	mask = /obj/item/clothing/mask/breath
+	head = /obj/item/clothing/head/helmet/space/plasmaman/janitor
+	uniform = /obj/item/clothing/under/plasmaman/janitor
+	gloves = /obj/item/clothing/gloves/color/plasmaman/janny
+
+/datum/outfit/plasmaman/party_bouncer
+	name = "ERP Bouncer Plasmaman"
```
Comment (ShizCalev) [inline code/modules/clothing/outfits/plasmaman.dm:248]: 🤔

Comment (GuillaumePrata) [reply code/modules/clothing/outfits/plasmaman.dm:248]: Not sure if you are pointing out the name... I'm just following the human version
>/datum/outfit/centcom/ert/security/party
name = "ERP Bouncer"

**Comment by Mothblocks** [comment]
this feels like a stupid implementation, but i guess we'll kick that can until someone inevitably wants different race outfits

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is now above zero (2). Feel free to make Feature/Tweak/Balance PRs.

**Pull request merged**